misc/python/materialize/cloudtest/util/web_request.py

```diff
@@ -62,7 +63,9 @@ def get(
         def try_get() -> requests.Response:
             with verbose_http_errors():
                 headers = self._create_headers(self.auth)
-                response = requests.get(
+                s = requests.Session()
```

Comment:
Optional: Do you want to introduce this behavior also for the other methods (`post`, ...)?

Reply:
retrying non-get methods is not safe in general - get is defined to be idempotent, but things like post are not

Reply:
Well, we have some retry logic there as well (for `401` errors), but it's ok for me to focus on `get`; in particular, if this is where the pain is.